tests/unit/conftest.py

```diff
@@ -50,18 +50,29 @@ class Snap:
         present = False
 
         def __init__(self):
-            self.services = {"mysqlrouter-service": {"active": False}}
+            self.services = {
+                "mysqlrouter-service": {"active": False},
+                "mysqlrouter-exporter": {"active": False},
+            }
 
         def ensure(self, *_, **__):
             return
 
+        def set(self, *_, **__):
+            return
+
+        def unset(self, *_, **__):
+            return
+
         def start(self, services: list[str] = None, *_, **__):
             assert services == ["mysqlrouter-service"]
             self.services["mysqlrouter-service"]["active"] = True
+            self.services["mysqlrouter-exporter"]["active"] = True
 
         def stop(self, services: list[str] = None, *_, **__):
             assert services == ["mysqlrouter-service"]
             self.services["mysqlrouter-service"]["active"] = False
+            self.services["mysqlrouter-exporter"]["active"] = True
```

Comment:
False?

Reply:
addressed in efea8849e6639f1fe0222a0832f699de0c7fc1ea